apps/src/maze/mazeMap.js

```diff
@@ -1,15 +1,63 @@
-class MazeMap {
+module.exports = class MazeMap {
```

Comment:
Why the switch from `export default` to `module.exports = `?

Reply:
for consistency with the eventual move to an extracted repo; after talking over the options with Josh, we've decided not to support ES6 import/export syntax in the extracted apps